tests/test_fs_utils.py

```
@@ -0,0 +1,43 @@
+        assert written.is_file()
+        assert written.read_bytes() == b"---\n- hosts: all\n"
+    finally:
+        import shutil
+
+        shutil.rmtree(tmp, ignore_errors=True)
+
+
+def test_write_chunked_fs_rejects_absolute_path() -> None:
+    """Absolute paths are rejected."""
+    files = [File(path="/etc/passwd", content=b"x")]
+    with pytest.raises(ValueError, match="Unsafe file path rejected|Path escapes temp root"):
```

Ruff (0.16.0)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)